### DISCUSSION TITLE
Please convert to V2 a very useful and amazing script

### DISCUSSION THREAD

**Discussion by H5820121** [mmikeww/AHK-v2-script-converter#318] [Q&A / Conversion help]
Can someone please convert this script to V2?
https://github.com/telppa/BeautifulToolTip

Thank you, and I'm sure many others will thank him too!

**Comment by Banaanae** [comment]
I converted this to a discussion because it isnt about a specific issue, ill take a look soon :)

**Comment by H5820121** [comment]
@Banaanae 
Now I saw that someone had already converted it:
https://github.com/liuyi91/ahkv2lib-
Thank you!
